scripts/inner_source_chroot_update.sh

```diff
@@ -6,125 +6,149 @@ TARGET="${1}"
 . /etc/profile
 
 # Setup environment vars
-export ETP_NONINTERACTIVE=1
+#export ETP_NONINTERACTIVE=1
 if [ -d "/usr/portage/licenses" ]; then
-	export ACCEPT_LICENSE="$(ls /usr/portage/licenses -1 | xargs)"
+  export ACCEPT_LICENSE="$(ls /usr/portage/licenses -1 | xargs)"
```

Comment:
This i think now can be safely replaced with `ACCEPT_LICENSE=*`

Reply:
Ok, I fix this.